docusaurus/sidebars.js

```diff
@@ -335,6 +335,7 @@ const sidebars = {
             'dev-docs/plugins/sentry',
             'dev-docs/plugins/upload',
             'dev-docs/plugins/users-permissions',
+            'dev-docs/plugins/content-source-map',
```

Comment:
My reasoning was that you need to pay for this one, so I wanted it to be at the end of the list. But now that I double-think about it, and because we don't have badges for EE-only stuff in the TOC, indeed that's a good idea to have it respecting the alphabetical order. I'll update this, thank you!